slapper.go

```diff
@@ -283,7 +284,8 @@ func reporter(quit <-chan struct{}) {
 			fmt.Print("\033[H") // clean screen
 			fmt.Printf("sent: %-6d ", sent)
 			fmt.Printf("in-flight: %-2d ", sent-recv)
-			fmt.Printf("\033[96mrate: %4d RPS\033[0m ", currentRate.Load())
+			fmt.Printf("requested rate: %4d RPS ", requestedRate.Load())
+			fmt.Printf("\033[96mresponse rate: %4d RPS\033[0m ", currentRate.Load())
```

Comment:
IMO requested/response rate are misleading names. Or maybe rather say "requested" rate can be misread as "request" rate. I suggest desired rate, current rate.

Also, "requested rate:  %d RPS, response rate: %d RPS" takes lots of space on a line. Keep in mind that if a text on a line would take more then screen width this will ruin histogram below. I suggest just "rate: %d/%d RPS" where first number is current rate, second number is desired rate.